controllers/vmware/vspherecluster_reconciler.go

```diff
@@ -295,25 +278,21 @@ func (r *ClusterReconciler) reconcileAPIEndpoints(ctx context.Context, clusterCt
 
 		// Only machines with bootstrap data will have an IP address.
 		if machine.Spec.Bootstrap.DataSecretName == nil {
-			log.V(5).Info(
-				"skipping machine while looking for IP address",
-				"reason", "bootstrap.DataSecretName is nil")
+			log.V(4).Info("Skipping machine while looking for IP address", "reason", "bootstrap.DataSecretName is nil")
```

Comment:
nit: `machine` and `Machine` are interchangibly used for log messages, is this intended?

Reply:
Not really. I just didn't fix it up consistently